week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -5,7 +5,17 @@ Write a function that:
 - removes any forward slashes (/) in the strings
 - makes the string all lowercase
 */
-function tidyUpString(strArr) {}
+function tidyUpString(strArr) {
+  let regex = /\//g;
+  // strArr[0] = strArr[0].replace(regex, "");
+  // console.log(arr)
+  let arr1 = [];
+  for (let i = 0; i < strArr.length; i++) {
+    arr1.push(strArr[i].replace(regex, "").trim().toLowerCase());
```

Comment:
Your solution is correct, however, would you like to give more explanation what is regex variable does here? (it's correct, but want to know the why)
and how Replace works?